guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc

```diff
@@ -37,6 +37,11 @@ For more information, see {InstallingServerDocURL}Preparing_your_Environment_for
 For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
 * Plan for updating any scripts you use that contain {Project} API commands because some API commands differ between versions of {Project}.
 
+ifdef::satellite[]
+NOTE: {Project} {ProjectVersion} is released only on {EL} 8. You must upgrade {Project} {ProjectVersionPrevious} from {EL} 7 to {EL} 8.
+For more information on upgrading from {EL} 7 to {EL} 8, see xref:upgrading-{project-context}-or-proxy-in-place-using-leapp_{context}[].
```

Comment:
```suggestion
NOTE: {Project} {ProjectVersion} is released only on {EL} 8.
You must upgrade {Project} {ProjectVersionPrevious} from {EL} 7 to {EL} 8.
For more information on upgrading from {EL} 7 to {EL} 8, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.11/html/upgrading_and_updating_red_hat_satellite/upgrading-satellite-or-proxy-in-place-using-leapp_upgrade-guide[Upgrading Satellite or Capsule to {RHEL} 8 In-Place Using Leapp].
```
[Ewoud's comment](https://github.com/theforeman/foreman-documentation/pull/1720#discussion_r1010600337) is indeed correct. If there are no procedures to upgrade from el 7 to 8 for satellite 6.12 then a link for 6.11 has to be included. @evgeni or @ekohl please confirm these changes.